packages/enzyme-matchers/src/utils/html.js

```diff
@@ -1,19 +1,26 @@
 import instance from './instance';
 /* eslint-disable no-console */
+let consoleObject;
+try {
+  consoleObject = console;
+} catch (e) {
+  // If no global console object is available, set consoleObject to a dummy object.
```

Comment:
My bad, I didn't see your associated issues until afterward. Could we maybe move this into it's own file so we can know to reference that in other places if we want to log things?